content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +151,44 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The use of sysctl in pods can be controlled through `forbiddenSysctls` and
+`allowedUnsafeSysctls` fields on the PodSecurityPolicy.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+By default, all safe sysctls are allowed. Currently, the whitelist of safe
+sysctls corresponds to the safe set mentioned above.
```

Comment:
will this ever change? The safe set above is the whitelist, isn't it?

Reply:
Yeah, it's the whitelist.
Dunno, later we can add additional field (e.g. `allowMoreSafeSysctls`) so the set of safe sysctls then does not have to correspond to the whitelist.